fmc_rest_client/resources/access_policy.py

```diff
@@ -86,3 +86,8 @@ def json_load(self, json):
             self.container = AccessPolicy()
             self.container.id = self.metadata.accessPolicy.id
 
+class IntrusionPolicy(PolicyResource):
+
+    def __init__(self, name=None):
+        super().__init__(name)
+        self.name = name
```

Comment:
You don't need to set self.name here as already you have set in super constructor or __init__ method.